<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Broadcast</class>
 <widget class="QGroupBox" name="Broadcast">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>270</width>
    <height>420</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>270</width>
    <height>420</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>270</width>
    <height>420</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>270</width>
    <height>420</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Jura</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Broadcast</string>
  </property>
  <widget class="QLabel" name="LabelURL">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>271</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Jura</family>
     <pointsize>9</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>http://servidor:puerto/mountpoint.extension</string>
   </property>
   <property name="textFormat">
    <enum>Qt::PlainText</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="LabelStatus">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>50</y>
     <width>271</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:#ff0000;</string>
   </property>
   <property name="text">
    <string>STATUS: OFFLINE </string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="LabelData">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>251</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Información: </string>
   </property>
  </widget>
  <widget class="QPushButton" name="ButtonRefrescar">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>390</y>
     <width>251</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Refrescar Información</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="TableInformacion">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>256</width>
     <height>271</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>250</width>
     <height>0</height>
    </size>
   </property>
   <property name="gridStyle">
    <enum>Qt::DotLine</enum>
   </property>
   <property name="rowCount">
    <number>10</number>
   </property>
   <property name="columnCount">
    <number>2</number>
   </property>
   <attribute name="horizontalHeaderCascadingSectionResizes">
    <bool>false</bool>
   </attribute>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>105</number>
   </attribute>
   <attribute name="horizontalHeaderMinimumSectionSize">
    <number>20</number>
   </attribute>
   <attribute name="horizontalHeaderStretchLastSection">
    <bool>true</bool>
   </attribute>
   <attribute name="verticalHeaderCascadingSectionResizes">
    <bool>true</bool>
   </attribute>
   <attribute name="verticalHeaderDefaultSectionSize">
    <number>30</number>
   </attribute>
   <attribute name="verticalHeaderMinimumSectionSize">
    <number>30</number>
   </attribute>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <column>
    <property name="text">
     <string>Nombre</string>
    </property>
    <property name="font">
     <font>
      <family>Jura</family>
      <pointsize>10</pointsize>
     </font>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Valor</string>
    </property>
    <property name="font">
     <font>
      <family>Jura</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
   </column>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
